docs/docs/developers/sdk/create-audius-app-react.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: create-audius-app-react
+title: Getting Started
+pagination_label: Create Audius App
+sidebar_label: Create Audius App
+description: Audius Protocol Documentation
+toc_min_heading_level: 2
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+Create Audius App is an officially supported way to create single-page Audius applications. It
+offers a modern build setup with no configuration.
+
+## Quick Start
+
+```sh
+npx create-audius-app
+cd my-app
+npm run dev
+```
+
+---
+
+## Creating an App
+
+The easiest way to start building on Audius is by using `create-audius-app`. This CLI tool enables
```

Comment:
Maybe we say something like "building a new application on top of the Audius Protocol"